node/pkg/driver/node.go

```diff
@@ -50,27 +48,33 @@ var (
 	}
 
 	defaultFSType              = "ext4"
-	stageInfoFilename          = ".stageInfo.json"
+	StageInfoFilename          = ".stageInfo.json"
 	supportedConnectivityTypes = map[string]bool{
 		"iscsi": true,
 		"fc":    true,
 		// TODO add nvme later on
 	}
 
 	IscsiFullPath = "/host/etc/iscsi/initiatorname.iscsi"
+
+	errorNoList = [...]int{53, 54, 59, 64, 65, 66, 67, 1219, 1326}
 )
 
 const (
-	// In the Dockerfile of the node, specific commands (e.g: multipath, mount...) from the host mounted inside the container in /host directory.
-	// Command lines inside the container will show /host prefix.
-	PrefixChrootOfHostRoot = "/host"
 	FCPath                 = "/sys/class/fc_host"
 	FCPortPath             = "/sys/class/fc_host/host*/port_name"
 )
 
+//go:generate mockgen -destination=../../mocks/mock_NodeMounter.go -package=mocks github.com/ibm/ibm-block-csi-driver/node/pkg/driver NodeMounter
+
+type NodeMounter interface {
+	mount.Interface
+	FormatAndMount(source string, target string, fstype string, options []string) error
+}
+
 // nodeService represents the node service of CSI driver
 type NodeService struct {
-	mounter                     *mount.SafeFormatAndMount
+	Mounter                     NodeMounter
```

Comment:
Impossible since we cannot chaange test package from driver_test to driver

Reply:
Please don't expose any internal attributes.

Reply:
Lue there were already other exposed in order unitests to work

Reply:
I assume the problem is because we create mock automatically which cause cyclic dependency if we put test file in the same package as testes file (e.g. and node.go and node_test.go).
Currently they are  in different packages (same folder but different packages)
We'll discuss if the change is crucial and if so will open separate ticket for it.

Reply:
The problem is that we group all the mocks in one folder, which is not a good practice. I remember I raised this issue (circular reference) in ubiquity project, but it is ignored.
It is important, you should not expose any internal attributes, please open ticket and attach the ticket number here before you resolve this conversation.

Reply:
please don't mark this conversation as resolved, it is not resolved yet.